assets/app/views/browse/deployment-config.html

```diff
@@ -105,24 +105,16 @@
             <td data-title="Deployment">
               <!-- Deployment number and link -->
               <span ng-if="deployment | annotation : 'deploymentVersion'">
-                <a ng-href="project/{{deployment.metadata.namespace}}/browse/deployments/{{deploymentConfig.metadata.name}}/{{deployment.metadata.name}}">#{{deployment | annotation : 'deploymentVersion'}}</a>
+                <a ng-href="{{deployment | navigateResourceURL}}">#{{deployment | annotation : 'deploymentVersion'}}</a>
                 <span ng-if="deploymentConfig.status.latestVersion == (deployment | annotation : 'deploymentVersion')">(latest)</span>
               </span>
             </td>
             <td data-title="Status">
               <div row class="status">
                 <!-- Deployment status -->
-                <span ng-switch="deployment | deploymentStatus" class="hide-ng-leave status-icon">
-                  <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>              
-                  <span ng-switch-when="Deployed" class="fa fa-check text-success" aria-hidden="true"></span>
-                  <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true"></span>
-                  <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true"></span>
-                  <span ng-switch-when="New" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
-                  <span ng-switch-when="Pending" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
-                  <span ng-switch-default class="fa fa-refresh" aria-hidden="true"></span>
-                </span>
+                <status-icon status="deployment | deploymentStatus"></status-icon>
```

Comment:
Are we sure `status-icon` handles all of the statuses for all types? Also check that you haven't started animating running icons where they weren't animated before.